web/controllers/respondent_controller.ex

```diff
@@ -629,7 +629,7 @@ defmodule Ask.RespondentController do
         |> Enum.map(fn mode -> mode_label([mode]) end)
         |> Enum.join(", ")
 
-        row = row ++ [modes]
+        row = row ++ [modes] ++ [Respondent.show_section_order(respondent.section_order, respondent, questionnaires)]
```

Comment:
Both lists can be collapsed into one to avoid one concatenation: `row = row + [modes, Respondent.show...]`

Also, the function always receives the `section_order` from the `respondent` and the `respondent` itself. So, actually only the `respondent` could be passed and the function should extract the `section_order` from it.